.github/workflows/ci.yml

```diff
@@ -17,6 +17,23 @@ jobs:
     - name: Check Formatting
       run: cargo fmt -- --check
 
+  typos:
+    name: Check for typos
+    runs-on: ubuntu-latest
+    timeout-minutes: 30
+    steps:
+      - uses: actions/checkout@v4
+      - name: Check for typos
+        uses: crate-ci/typos@v1.18.2
```

Comment:
I would prefer if we stick to the known [`taiki-e/install-action`](https://github.com/taiki-e/install-action) instead.

Reply:
Ha, I didn't realize that the `crate-ci/typos` action is the same repo as the `typos` project itself.
Well, its fine either way.